sortinghat/core/api.py

```diff
@@ -1375,3 +1377,93 @@ def update_import_identities_task(ctx, task_id, **kwargs):
     logger.info(f"Task {task_id} successfully updated")
 
     return task
+
+
+@atomic_using_tenant
+def merge_organizations(ctx, from_org, to_org):
+    """Merge an organization into another.
+
+    This function moves the domains, teams and affiliations from
+    an origin organization identified by `from_org` to the target
+    organization identified by `to_org`. The origin organization is
+    then deleted.
+
+    The function raises a `NotFoundError` exception when either `from_org`
+    or `to_org` do not exist in the registry and both identifiers are
+    not the same. If both are the same, it raises an `InvalidValueError`.
+
+    The target organization with updated data is returned as a result.
+
+    :param ctx: context from where this method is called
+    :param from_org: identifier of the origin organization
+    :param to_org: identifier of the target organization where the origin
+        organization will be merged
+
+    :returns: an organization with the domains, teams and affiliation
+        data updated
+
+    :raises NotFoundError: raised when either `from_org` or `to_org`
+        do not exist in the registry.
+    :raises InvalidValueError: raised when either `from_org' or `to_org`
+        are `None` or empty strings, or their values are the same.
+    """
+    def _move_domains(trxl, from_org, to_org):
+        """Move the domains from the origin organization to the target"""
+        domains = from_org.domains.all()
```

Comment:
In GrimoireLab when there is a multi-line docstring we don't leave an empty line after it, but when the docstring is a single line we add it.

```suggestion

        domains = from_org.domains.all()
```